tests/progaudi.tarantool.tests/Schema/GetSpace_Should.cs

```diff
@@ -81,5 +82,17 @@ public class GetSpace_Should : TestBase
                 }
             }
         }
+
+        [Fact]
+        public async Task iterating_over_schema()
+        {
+            using (var tarantoolClient = await Client.Box.Connect(ConnectionStringFactory.GetReplicationSource_1_7()))
+            {
+                foreach (var pair in (IReadOnlyDictionary<string, ISpace>)tarantoolClient.Schema)
```

Comment:
How about exposing IndexByName and IndexById as IReadonlyDictionaries instead of decoration?

Reply:
`box.Schema.IndexByName` ?

Reply:
Sorry, typo, should be:
* Properties IndexByName and IndexById in Space 
* Properties SpaceByName and SpaceById in Schema